syntax = "proto3";

package astre.ecs;

message TerrainComponent 
{
  uint32 width = 1;
  uint32 length = 2;
  float tile_size = 3;
}